dts/arm/ti/j722s_mcu_r5.dtsi

```diff
@@ -58,4 +58,14 @@
 		interrupt-parent = <&vim>;
 		status = "disabled";
 	};
+
+	mbox1: mailbox@29010000 {
```

Comment:
@glneo should we start giving node labels to mcu peripherals with a mcu prefix instead? mcu_mboxN instead?

Reply:
Yup, that would be great. The more similar the dt is with Linux, the better.